src/styles/SwitchStyle.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const SwitchLabel = styled.label`
+  display: inline-flex;
+  align-items: center;
+  justify-content: space-between;
+  cursor: pointer;
+  width: 46px;
+  height: 26px;
+  border-radius: 100px;
+  border: 2px solid gray;
+  position: relative;
+  transition: background-color 0.2s;
+
+  @media (max-width: 800px) {
+    visibility: hidden;
+  }
+
+  div {
+    margin-left: -100px;
+  }
+`;
+
+const SwitchInput = styled.input`
+  height: 0;
+  width: 0;
+  visibility: hidden;
+`;
+
+const SwitchButton = styled.span`
```

Comment:
Awesome, you can improve this by moving the styled components to their own file, something like `Switch.styled.jsx` can contain only the styled components